FAQ.md

```diff
@@ -0,0 +1,251 @@
+
+---
+
+### **Q: Why was CDEvents created?**
+**A:** CDEvents was created to address the fragmentation in the Continuous Delivery (CD) toolchain. By providing a common event specification, it allows different tools and platforms to communicate more effectively, reducing integration complexity and preventing vendor lock-in. This fosters a more flexible and adaptable CD pipeline.
+
+---
+
+### **Q: Who is behind CDEvents?**
+**A:** CDEvents is developed and maintained by a collaborative working group comprising industry experts, open-source contributors, and representatives from various organizations committed to enhancing Continuous Delivery practices.
+
+---
+
+### **Q: How does CDEvents differ from other event specifications?**
+**A:** Unlike proprietary event formats or tool-specific integrations, CDEvents is an open, vendor-neutral specification focused solely on Continuous Delivery events. It emphasizes interoperability, flexibility, and ease of adoption across diverse toolchains and environments.
+
+---
+
+## Technical Questions
+
+### **Q: How does CDEvents ensure event consistency across tools?**
+**A:** CDEvents provides a standardized event schema that all compliant tools must follow. This includes consistent naming conventions, required fields, and data formats. By adhering to this schema, events emitted by different tools maintain uniformity, simplifying integration and processing.
+
+---
+
+### **Q: What event types are defined by CDEvents?**
+**A:** CDEvents defines a variety of event types covering different stages of the CD pipeline, including but not limited to:
+- **Build Events:** Indicate the start, success, or failure of a build process.
+- **Test Events:** Capture test execution details and results.
+- **Deploy Events:** Represent deployment actions, including initiation and completion.
+- **Pipeline Events:** Provide insights into the overall workflow and pipeline status.
+- **Approval Events:** Handle manual approval steps within the pipeline.
+- **Artifact Events:** Manage artifact creation, storage, and retrieval.
+
+For a comprehensive list, refer to the [CDEvents specification](https://github.com/cdevents/spec/blob/main/spec.md).
+
+---
+
+### **Q: What technologies does CDEvents use?**
+**A:** CDEvents is technology-agnostic and can be implemented using various protocols and data formats. Common implementations use JSON or YAML for event payloads and can be transported over HTTP, WebSockets, or message brokers like Kafka and RabbitMQ.
+
+---
+
+### **Q: Are there any libraries or SDKs available for CDEvents?**
+**A:** Yes, the community has developed several libraries and SDKs in various programming languages to facilitate the implementation of CDEvents. You can find these resources in our [GitHub repositories](https://github.com/cdevents/spec) or [Docs](https://cdevents.dev/docs/) or explore community-contributed projects.

```

Comment:
I think it would be good to provide links to SDK repos, like https://github.com/cdevents/sdk-go, https://github.com/cdevents/sdk-java and https://github.com/cdevents/sdk-rust. We could also mention that more SDKs are being developed and that contributions are welcome

Reply:
### **Q: Are there any libraries or SDKs available for CDEvents?**
**A:** Yes, the community has developed several libraries and SDKs in various programming languages to facilitate the implementation of CDEvents. Below are some of the available SDKs:

- [Go SDK](https://github.com/cdevents/sdk-go)
- [Java SDK](https://github.com/cdevents/sdk-java)
- [Rust SDK](https://github.com/cdevents/sdk-rust)

More SDKs are being developed, and contributions are welcome! You can find additional resources in our [GitHub repositories](https://github.com/cdevents/spec) or in the [official documentation](https://cdevents.dev/docs/).


Would this work ?